packages/avro-cpp/build

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export CPPFLAGS=-I/opt/mesosphere/include
+export LDFLAGS="-L/opt/mesosphere/lib -Wl,-rpath=/opt/mesosphere/lib"
+
+pushd /pkg/src/avro-cpp
+mkdir -p build
+pushd build
+
+# configure build with cmake
+# see https://cmake.org/cmake/help/v3.0/module/FindBoost.html
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=$PKG_PATH \
+  -DBOOST_ROOT=/opt/mesosphere/ \
+  ..
+
+# build/unittest
+make -j8
```

Comment:
This needs to use `make -j$NUM_CORES` (see: https://github.com/dcos/dcos/commit/c55b0c82c3c74d1e5c37b7d944d527bcda6e5dd9)